pkg/api/devops/v1alpha3/template_types.go

```diff
@@ -48,6 +48,10 @@ type TemplateParameter struct {
 	//+optional
 	Description string `json:"description,omitempty"`
 
+	// Required indicates if this parameter is mandatory.
+	//+optional
+	Required bool `json:"required"`
```

Comment:
I'm wondering if this field needs `omitempty` or not. What do you think about it?

Reply:
I agree with you. Good catch!